examples/jsm/nodes/core/PropertyNode.js

```diff
@@ -55,5 +55,6 @@ export const shininess = nodeImmutable( PropertyNode, 'float', 'Shininess' );
 export const output = nodeImmutable( PropertyNode, 'vec4', 'Output' );
 export const dashSize = nodeImmutable( PropertyNode, 'float', 'dashSize' );
 export const gapSize = nodeImmutable( PropertyNode, 'float', 'gapSize' );
+export const pointWidth = nodeImmutable( PropertyNode, 'float', 'pointWidth' );
```

Comment:
Please add this to the exports in Nodes.js.

Reply:
Here I meant adding this new export also to the list of all PropertyNode's exports in Nodes.js: `export { default as PropertyNode, property, output, diffuseColor, roughness, metalness, specularColor, shininess } from './core/PropertyNode.js'` (seems like the latest exports were also not added there)

Reply:
I will just file a new PR for this after this one is merged.

Reply:
OK, resolving for now.